swagger.yml

```diff
@@ -1653,6 +1657,10 @@ paths:
         name: amount
         type: integer
         default: 100
+      - in: query
+        name: type
+        type: string
+        <<: *DIFF_TYPE
```

Comment:
I don't understand the `<<: *...` notation :(

Reply:
This is a swagger 2 syntax that allows you to reuse enum types.
In OpenAPI 3 they are reusable in a more straightforward way.
 
For more information: https://swagger.io/docs/specification/2-0/enums/